wdn/templates_4.1/scripts/unlalert.js

```diff
@@ -178,26 +192,35 @@ define([
 
 			web = info[i].web || 'http://www.unl.edu/';
 
-			alertContentHTML = '<h1><span>Emergency UNL Alert:</span> ' + info[i].headline + '</h1>';
-			alertContentHTML += '<h2>Issued at ' + effectiveDate + '</h2>';
-			alertContentHTML += '<p>' + info[i].description + '<br/>';
+			alertContentHTML = '<div class="unlalert-info"><div class="wdn-sans-caps unlalert-heading">Emergency alert</div><div class="wdn-impact unlalert-headline">' + info[i].headline + '</div><p class="unlalert-desc">' + info[i].description + '</p>';
 			if (info[i].instruction) {
-				alertContentHTML += info[i].instruction + '<br/>';
+				alertContentHTML += '<p class="unlalert-desc">' + info[i].instruction + '</p>';
 			}
-			alertContentHTML += 'Additional info (if available) at <a href="' + web + '">' + web + '</a></p>';
+			alertContentHTML += '</div><div class="unlalert-meta"><div class="unlalert-datetime"><div class="wdn-sans-caps unlalert-heading">Issued</div><div>' + effectiveDate + '</div></div><div class="unlalert-link"><div class="wdn-sans-caps unlalert-heading">Additional info (if available)</div><div><a href="' + web + '">' + web + '</a></div></div></div>';
```

Comment:
While I agree, my rationale was that because changes to UNL Alert are infrequent, we could save some bytes by avoiding multiple lines of `alertContentHTML +=`